db/migrate/20180913215015_change_culled_to_status.rb

```diff
@@ -0,0 +1,18 @@
+class ChangeCulledToStatus < ActiveRecord::Migration
+  def change
+    reversible do |change|
+      change.up{
+        rename_column :games, :culled, :status
```

Comment:
This seems like a pretty dangerous way to do this, maybe make a new column and delete the old one?

Reply:
I completely agree, if this was running in production and needed to be up all the time. But I think this is fine for our use case.

Reply:
Have you run it as a test on our actual db and made sure things looked right? If so, I agree it's good enough.

Reply:
Yup, I'm testing it on a full copy of the program from West.